specs/protocol/configurability.md

```diff
@@ -0,0 +1,76 @@
+| Sequencing window                     | Maximum allowed batch submission gap, after which L1 fallback is triggered in derivation.                                                                                                                                                            |
+| Start block                           | Block at which the system config was initialized the first time.                                                                       |
+
+## Admin Roles
+
+| Config Property                       | Description                                                                                                                  |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| L1 Proxy Admin                        | Account authorized to upgrade L1 contracts.                                                                                  |
+| L1 ProxyAdmin owner                   | Account authorized to update the L1 Proxy Admin.                                                                             |
+| L2 Proxy Admin                        | Account authorized to upgrade L2 contracts.                                                                                  |
+| L2 ProxyAdmin owner                   | Account authorized to update the L2 Proxy Admin.                                                                             |
+| System Config Owner                   | Account authorized to change values in the SystemConfig contract.                                                            |
+
+## Service Roles
+
+| Config Property                       | Description                                                                                                                  |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| Batch submitter address               | Account which authenticates new batches submitted to L1 Ethereum.                                                            |
+| Challenger address                    | Account which can delete output roots before challenge period has elapsed.                                                   |
+| Guardian address                      | Account authorized to pause L1 withdrawals from contracts.                                                                   |
+| Proposer address                      | Account which can propose output roots to L1.                                                                                |
+| Sequencer P2P / Unsafe head signer    | Account which authenticates the unsafe/pre-submitted blocks for a chain at the P2P layer.                                    |
+## Policy Parameters
```

Comment:
@tynes new table for policy params

Reply:
It is important to clarify these as the things that can be changed by anybody and don't impact standardness